shared/transforms/shared_constants.xslt

```diff
@@ -13,7 +13,7 @@
 <xsl:variable name="cnss1253uri">http://www.cnss.gov/Assets/pdf/CNSSI-1253.pdf</xsl:variable>
 <xsl:variable name="dcid63uri">not_officially_available</xsl:variable>
 <xsl:variable name="disa-cciuri">http://iase.disa.mil/stigs/cci/Pages/index.aspx</xsl:variable>
-<xsl:variable name="disa-srguri">http://iase.disa.mil/stigs/srgs/Pages/index.aspx</xsl:variable>
+<xsl:variable name="disa-srguri">https://iase.disa.mil/stigs/os/general/Pages/index.aspx/</xsl:variable>
```

Comment:
This change breaks OpenSCAP HTML report. I agree we need to do it but it definitely needs to be a separate PR with tracking to the OpenSCAP change. Please don't clump multiple changes into one PR.

Reply:
@mpreisler I think we should think about making OpenSCAP HTML report more clever. This affects us each time when some URL changes.

Reply:
I agree, the report should recognize new and old URLs to be compatible with a wide variety of existing SCAP content.